install.sh

```diff
@@ -733,6 +783,37 @@ service_enable_and_start() {
     return 0
 }
 
+# --- create symlink to kubeconfig
+setup_kubeconfig() {
+    DEFAULT_CONFIG=${HOME}/.kube/config
+    if [ ! -f "${KUBECONFIG}" ]; then
+        info "Waiting for kubeconfig at ${KUBECONFIG}..."
+        for _ in $(seq 1 20); do
+            [ -f "${KUBECONFIG}" ] && break
+            sleep 1
+        done
+        [ -f "${KUBECONFIG}" ] || warn "Kubeconfig does not yet exist at ${KUBECONFIG}"
+    fi
+
+    if [ -L ${DEFAULT_CONFIG} ]; then
+        CONFIG_LINK=$(readlink -f ${DEFAULT_CONFIG} || true)
+        if [ "${CONFIG_LINK}" == "${KUBECONFIG}" ] || echo ${CONFIG_LINK} | grep -vq k3s; then
```

Comment:
Better use = than == since the POSIX single = is what the existing k3s scripts (almost) consistently use.